ansible/deploy-all.yaml

```diff
@@ -1,35 +1,38 @@
 ---
 ############################################
+# Third-party
+############################################
+- include: infra-amqp-brokers.yaml
+  when: not parasitic
+
+- include: infra-deploy-irods-elasticsearch.yaml
+  when: not parasitic
+
+############################################
```

Comment:
Nuked the third party infra playbooks from deploy-all.yaml. Working on cleaning up the rest.